internal/builders/docker/pkg/options.go

```diff
@@ -37,4 +38,7 @@ func (io *InputOptions) AddFlags(cmd *cobra.Command) {
 
 	cmd.Flags().StringVarP(&io.BuilderImage, "builder-image", "i", "",
 		"Required - URL indicating the Docker builder image, including a URI and image digest.")
+
+	cmd.Flags().BoolVarP(&io.ForceCheckout, "force-checkout", "f", false,
```

Comment:
this can go away, right?

Reply:
Good catch. Thanks.

Reply:
just this one too!

Reply:
ah wait, this is reintroduced after removing the 3 val - resolving